corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js

```diff
@@ -534,3 +538,30 @@ WebFormSession.prototype.renderFormXml = function (resp, $form) {
     self.session_id = self.session_id || resp.session_id;
     self.form = Formplayer.Utils.initialRender(resp, self.resourceMap, $form);
 };
+
+WebFormSession.prototype.reportFormplayerErrorToHQ = function (resp, action, errorMessage) {
+    var self = this;
+    var data = {
+        action: action,
+        readableErrorMessage: errorMessage,
+        statusText: resp.statusText,
+        state: resp.state ? resp.state() : null,
+        status: resp.status,
+        domain: self.domain,
```

Comment:
Nitpick: is domain needed, given that it's also embedded in the URL?

Reply:
Maybe not, but my thought was that it doesn't hurt. This will make it show up in the sentry "additional data" section, which may or may not be useful given it's other places too